---
title: LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER | Документация Майкрософт
ms.custom: ''
ms.date: 03/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: c4288265-43f7-44fd-acab-ba64d04264d9
author: stevestein
ms.author: sstein
ms.openlocfilehash: f10ed404b9ccb2d7a1dd62a45ac794df265a45d9
ms.sourcegitcommit: da88320c474c1c9124574f90d549c50ee3387b4c
ms.translationtype: MT
ms.contentlocale: ru-RU
ms.lasthandoff: 07/01/2020
ms.locfileid: "85635317"
---
# <a name="localdb_error_cannot_access_instance_folder"></a>LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER
 [!INCLUDE [SQL Server](../../includes/applies-to-version/sqlserver.md)]
    
## <a name="details"></a>Сведения  
  
|||  
|-|-|  
|Название продукта|SQL Server|  
|Идентификатор события|261|  
|Источник события|Среда выполнения локальной базы данных SQL Server 12.0|  
|Компонент|API среды выполнения локальной базы данных|  
|Текст сообщения|Не удается получить доступ к папке экземпляра локальной базы данных:%% LOCALAPPDATA%% \ Microsoft\Microsoft SQL Server локальное \\ имя экземпляра дб\инстанцес<\> .|  
  
## <a name="explanation"></a>Объяснение  
 Не удалось получить доступ к папке экземпляра.  
  
## <a name="user-action"></a>Действие пользователя  
  
